config/services_tests_config_prod.json

```diff
@@ -18,5 +18,6 @@
     "subset-band-name": "subset-band-name",
     "swath-projector": "swath-projector",
     "trajectory-subsetter": "nsidc-icesat2,trajectory-subsetter",
+    "sambah": "sambah",
```

Comment:
`sambah` is a service chain, not a single service. So we should not add a `sambah` service here, but add the `sambah` test in tests of batchee and stitchee services. .i.e. change `TBD` on line 3 and line 17 to `sambah`. 

The same applies to the UAT configuration.

Reply:
oh I see, thanks @ygliuvt!